### PR TITLE
Sj branch

### PR DESCRIPTION
**Pull request by seejie25**

### CODE REVIEW

File: pipeline/sj_luigi_luigi.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def run(self):
         # creates EmailMessage object
         msg = EmailMessage()
         msg["Subject"] = self.subject
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
```
Comment (geraldooi) [inline pipeline/sj_luigi_luigi.py:113]: Can you explain to me why both of you using the same -+?

File: pipeline/sj_luigi_luigi.py [hunk 2]
```diff
@@ -110,8 +110,9 @@ def run(self):
...
         msg["From"] = email().email_from
-        msg["To"] = ", ".join(self.receipients)
+        msg["To"] = email().email_to.split(", ")
```
Comment (geraldooi) [inline pipeline/sj_luigi_luigi.py:115]: how you put your email_to?
email_to="email1@example.com, email2@example.com"
or
email_to=email1@example.com, email2@example.com

File: pipeline/sj_luigi_luigi.py (hunk 1 above)
Comment (seejie25) [reply pipeline/sj_luigi_luigi.py:113]: We discussed together yesterday so we are using the same code for this part. And regarding to the -+, I think it should be - only, that might be a typo, but the output is fine so both of us didn't realize that.

Comment (geraldooi) [reply pipeline/sj_luigi_luigi.py:113]: I see. OK.

File: pipeline/sj_luigi_luigi.py (hunk 2 above)
Comment (seejie25) [reply pipeline/sj_luigi_luigi.py:115]: the second one, email_to=[email1@example.com](mailto:email1@example.com), [email2@example.com](mailto:email2@example.com)

Comment (geraldooi) [reply pipeline/sj_luigi_luigi.py:115]: ok. actually you dont need to use split, what if someone else write it without space after 'comma', email_to=email1@example.com,email2@example.com

do you think your split still work?

Comment (geraldooi) [reply pipeline/sj_luigi_luigi.py:115]: if you used luigi.Parameter(), the luigi will treat it as str type, so you just need to msg["To"]=email().email_to will do.

Comment (geraldooi) [reply pipeline/sj_luigi_luigi.py:115]: no need to change anything

Comment (seejie25) [reply pipeline/sj_luigi_luigi.py:115]: Ok I will have a try, thanks

Comment (seejie25) [reply pipeline/sj_luigi_luigi.py:115]: Now I got it! Tysm

**Pull request merged**